src/TableListItem/TableListItem.st.css

```
@@ -0,0 +1,85 @@
+:import {
+  -st-from: "../Foundation/stylable/colors.st.css";
+  -st-named: B10, B30, B60, D50, D60, D80;
+}
+
+:vars {
+  horizontalPadding: 24px;
+  checkedBorderWidth: 2px;
+  smallPadding: 12px;
+  mediumPadding: 18px;
+}
+
+.root {
+  -st-states: checked, showDivider, smallVerticalPadding, mediumVerticalPadding, draggable;
+  background-color: value(D80);
+  padding: 0 value(horizontalPadding);
+}
+
+.root:checked {
+  border-left-width: value(checkedBorderWidth);
+  border-left-style: solid;
+  border-left-color: value(B10);
+  padding-left: calc(value(horizontalPadding) - value(checkedBorderWidth));
+  padding-right: value(horizontalPadding);
+}
+
+.root:showDivider {
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-bottom-color: value(D60);
+}
+
+.root:smallVerticalPadding {
+  padding-top: value(smallPadding);
+  padding-bottom: value(smallPadding);
+}
+
+.root:mediumVerticalPadding {
+  padding-top: value(mediumPadding);
+  padding-bottom: value(mediumPadding);
+}
+
+.root:draggable:hover {
+  background-color: value(B60);
+}
+
+.optionsContainer {
+  display: grid;
+  width: 100%;
+}
+
+.align {
+  -st-states: left, center, right;
```

You can use an enum:
```suggestion
  -st-states: align(enum(left, center, right));
```